content/router/lazy-load-feature-modules.md

```diff
@@ -41,7 +41,10 @@ This means that the `UsersModule` will be added to the main bundle. The main bun
 // app.routing.ts
 const routes: Routes = [
   ...
-  {path: 'users', loadChildren: '../users/usersModule#UserModule'}
+  {
+    path: 'users',
+    loadChildren: () => import('../users/usersModule').then(m => m.UsersModule)
+    }
```

Comment:
This formatting is still a bit off. The closing bracket is indented. Sorry for this tiny nit-pick again, but I wanna make sure content is properly formatted. I think you have to remove 2 spaces, but please double check.

Reply:
@d3lm You are right, i corrected the indentation.